app_noticias/views.py

```diff
@@ -17,6 +17,11 @@ class HomePageView(ListView):
     def get_queryset(self):
         return Noticia.objects.exclude(data_de_publicacao=None).order_by('-data_de_publicacao')[:5]
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['noticias_recentes'] = Noticia.objects.exclude(data_de_publicacao=None, categoria__nome='Destaque').order_by('-data_de_publicacao')[:5]
```

Comment:
excluir a notícia destaque não seria dessa forma. tem que identificar a notícia de destaque primeiro e, depois, excluí-la da lista. mais ou menos assim:

```python
noticia_destaque = Noticia.objects.filter(categoria__nome='Destaque').order_by('-data_de_publicacao').first()
noticias_recentes = Noticia.objects.exclude(id__in=[noticia_destaque.id]).order_by('-data_de_publicacao')[:5]
```